### PR TITLE
Removed Qt references from Libpfs

### PR DESCRIPTION
**Pull request by dnikishov**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -91,7 +91,7 @@ ENDIF()
 IF(WIN32)
     SET(Boost_USE_STATIC_LIBS ON)
 ENDIF()
-FIND_PACKAGE(Boost REQUIRED COMPONENTS program_options)
+FIND_PACKAGE(Boost REQUIRED COMPONENTS program_options system)
```
Comment (davideanastasia) [inline CMakeLists.txt]: Why is this change necessary?

Comment (dnikishov) [inline CMakeLists.txt]: It seems that Boost.Thread depends on Boost.System. Hence the explicit requirement for it.

**Comment by danielkaneider** [comment]
The original patch doesn't compile with VS. However, when I change CMake to 

`FIND_PACKAGE(Boost REQUIRED COMPONENTS program_options thread chrono system date_time)`

everything works fine

**Comment by danielkaneider** [comment]
Thanks for the contribution!
